transaction/transaction.go

```diff
@@ -326,15 +336,7 @@ func MakeAssetConfigTxn(account string, feePerByte, firstRound, lastRound uint64
 		Note:        note,
 	}
 
-	creatorAddr, err := types.DecodeAddress(creator)
-	if err != nil {
-		return tx, err
-	}
-
-	tx.ConfigAsset = types.AssetID{
-		Creator: creatorAddr,
-		Index:   index,
-	}
+	tx.ConfigAsset = types.AssetIndex(index)
 
 	if newManager != "" {
 		tx.Type = types.AssetConfigTx
```

Comment:
I believe this line is not necessary

Reply:
It is necessary - without it, compilation fails with `cannot use index (type uint64) as type types.AssetIndex in assignment`